### PR TITLE
[MIG] website_event_share: Migration to 10.0

### PR DESCRIPTION
**Pull request by chienandalu**
Event Share
===========

This module brings the users the possibility to globally enable/disable a share
links block that will be displayed in every event post.

cc @Tecnativa

### CODE REVIEW

File: website_event_share/README.rst [hunk 1]
```diff
@@ -6,27 +6,29 @@
...
    :alt: Try me on Runbot
-   :target: https://runbot.odoo-community.org/runbot/199/9.0
+   :target: https://runbot.odoo-community.org/runbot/199/10.0
 
 Know issues / Roadmap
 =====================
-* Share event configuration valid for all events. No individual configuration for each event
 
+* Share event configuration is set on all events. No individual configuration
+for each event
```
Comment (pedrobaeza) [inline website_event_share/README.rst]: Indent this 2 spaces.

**Review by pedrobaeza — APPROVED** [review]
Minor detail

**Comment by chienandalu** [comment]
@pedrobaeza Changes done

File: website_event_share/views/website_event.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+<odoo>
+    <template id="event_share_view" name="Share Button Snippet">
+        <xpath expr="//h4[@t-field='event.address_id']" position="after">
+            <div class="oe_share s_share text-center">
+                  <h3><span>Share </span>
+                    <a class="oe_share_facebook" t-attf-href="https://www.facebook.com/sharer/sharer.php?u={url}" target="_Blank"><i class="fa fa-facebook-square"/></a>
+                    <a class="oe_share_twitter" t-attf-href="https://twitter.com/intent/tweet?text={{event.website_meta_title}}&amp;url={url}" target="_Blank"><i class="fa fa-twitter"/></a>
+                    <a class="oe_share_linkedin" t-attf-href="http://www.linkedin.com/shareArticle?mini=true&amp;url={url}&amp;title={{event.website_meta_title}}" target="_Blank"><i class="fa fa-linkedin"/></a>
+                    <a class="oe_share_google" t-attf-href="https://plus.google.com/share?url={url}" target="_Blank"><i class="fa fa-google-plus-square"/></a>
+                    <a href="mailto:?body={url}&amp;subject={title}"><i class="fa fa-envelope-o"> </i></a>
```
Comment (yajo) [inline website_event_share/views/website_event.xml]: Any chances we can reuse [`website.s_share`](https://github.com/odoo/odoo/blob/668926ff4f75a66634991282ffabefc66aadf924/addons/website/views/snippets.xml#L267-L278) instead?

File: website_event_share/views/website_event.xml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="event_share_view" name="Share Button Snippet">
+        <xpath expr="//h4[@t-field='event.address_id']" position="after">
```
Comment (yajo) [inline website_event_share/views/website_event.xml]: How come you use xpath in a template without `inherit_id`?

**Review by yajo — CHANGES_REQUESTED** [review]
A couple of things to improve.

OTOH, are you really sure you want to maintain this addon? No offense, but Odoo 10 already has a "Share" snippet that does basically the same (well, you have to drag and drop once per event, but also lets the user choose where to put it):

captura de pantalla de 2017-06-23 09-19-00

IMHO I feel it's better to train users to use core tools than adding addons as this one.

**Comment by pedrobaeza** [comment]
@Yajo the module should be maintained to assure it always appear, but as you have said, maybe this can reuse some of the standard logic.

**Comment by chienandalu** [comment]
@Yajo Changes done

**Review by yajo — APPROVED** [review]
Code review seems OK.

File: website_event_share/views/website_event.xml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="event_share" name="Share Button" inherit_id="website_event.event_details" active="False" customize_show="True">
```
Comment (yajo) [inline website_event_share/views/website_event.xml]: Remove `active=False` to get the view enabled by default when installed.

**Review by yajo — CHANGES_REQUESTED** [review]

**Comment by chienandalu** [comment]
@Yajo Done

**Review by yajo — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@chienandalu squash your commits and I'll merge

**Comment by chienandalu** [comment]
@pedrobaeza Done

**Comment by yelizariev** [comment]
Is it correct description in manifest?

        'summary': 'Add contacts to event',

**Comment by pedrobaeza** [comment]
Obviously not (copy/paste error). Fixed in https://github.com/OCA/event/commit/72900bf4c21a3dd31144d58fe52f72ca70f1ec1b

**Comment by yelizariev** [comment]
@pedrobaeza you fixed in 2 minutes after reporting! Thank you :smile_cat:

**Pull request merged**